src/app/pages/MediaArticlePage/MediaArticlePage.styles.ts

```diff
@@ -86,4 +91,13 @@ export default {
         padding: `${spacings.DOUBLE}rem`,
       },
     }),
+
+  responsiveComponentWrapper: ({ mq }: Theme) =>
+    css({
+      marginBottom: `${GEL_SPACING_TRPL}`,
+      [mq.GROUP_4_MIN_WIDTH]: {
+        marginBottom: `${GEL_SPACING}`,
+        padding: `${GEL_SPACING_DBL}`,
+      },
```

Comment:
These `GEL_` spacings should have their equivalents available on `spacings` from the Theme, so we could use those and remove the Psammead import.

Reply:
Done